### PR TITLE
[WIP] feat: add travis

### PR DESCRIPTION
**Pull request by hugomrdias**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  include:
+    - stage: check
+      node_js: 10
+      script:
+        - npm run lint
+
+    - stage: test
+      name: "node-10"
+      node_js: 10
+      script: npm run test:node:http -- --bail
```
Comment (alanshaw) [inline .travis.yml]: Just `http`?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      os: osx
+      name: "osx-node-10"
+      node_js: 10
+      script: npm run test:node:http -- --bail
+
+    - stage: test
+      os: windows
+      name: "windows-node-10"
+      node_js: 10
+      script: npm run test:node:http -- --bail
```
Comment (alanshaw) [inline .travis.yml]: Needs browser, web worker and coverage steps!

File: .travis.yml [hunk 3]
```diff
@@ -1,26 +1,29 @@
 language: node_js
 cache: npm
+stages:
+  - check
+  - test
+  - cov
+
+node_js:
+  - 10
+  - 11
```
Comment (daviddias) [inline .travis.yml]: no need. Often the odd version is unstable and breaks things.

**Comment by alanshaw** [comment]
Any update on this @hugomrdias?

**Comment by hugomrdias** [comment]
Yes

**Pull request closed without merging**